Makefile

```diff
@@ -0,0 +1,14 @@
+PG_CONFIG ?= $(shell which pg_config)
+
+all: package
+
+# Install vchord into the PostgreSQL cluster identified by pg_config.
+.PHONY: install
+install:
```

Comment:
PGXN clients runs `sudo make install` to installs the client, and it causes build `./crates/make`.

Correct steps should be:

1. `make`

```
cargo build -p make
cp ./target/debug/make ./build/make
```

2. `build`

```
./build/make build -o ./build/raw
```

3. `install`

```
./build/make install -i ./build/raw
```

Reply:
```makefile
PG_CONFIG ?= $(shell which pg_config)

.PHONY: make package install
.DEFAULT_GOAL: package

all: build-make package

build-make:
	cargo build -p make
	cp ./target/debug/make ./build/make

package:
	PGRX_PG_CONFIG_PATH="$(PG_CONFIG)" ./build/make package -o ./build/raw

install:
	PGRX_PG_CONFIG_PATH="$(PG_CONFIG)" ./build/make install -i ./build/raw
```

Reply:
ok, fixed.